pydantic/env_settings.py

```diff
@@ -5,10 +5,12 @@
 
 from .fields import ModelField
 from .main import BaseConfig, BaseModel, Extra
+from .types import SecretBytes, SecretStr
 from .typing import display_as_type
 from .utils import deep_update, sequence_like
 
 env_file_sentinel = str(object())
+secrets_dir_sentinel = str(object())
```

Comment:
this doesn't seem to be used.

Reply:
Yep, leftover from copying the environment code. Will remove